contracts/MyToken_1155.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Supply.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+contract MyToken_1155 is ERC1155Supply, Ownable, ReentrancyGuard {
+
+    uint public tokenPriceByEth;
+    uint public tokenPriceByErc20;
+    uint public amountLimitOfToken;
+    uint public countOfSet;
+    string private _tokenBaseUri;
+
+    constructor(string memory baseUri, uint priceByEth, uint priceByErc20, uint countOfSetForToken, uint amountLimit) 
+        ERC1155("") 
```

Comment:
I believe `_tokenBaseUri` needs to be passed as an argument for `ERC1155` constructor.